patroni/postgresql/sync.py

```diff
@@ -201,35 +221,35 @@ def _handle_synchronous_standby_names_change(self) -> None:
 END;$$""")
         self._postgresql.reset_cluster_info_state(None)  # Reset internal cache to query fresh values
 
-    def current_state(self, cluster: Cluster) -> Tuple[CaseInsensitiveSet, CaseInsensitiveSet]:
-        """Finds best candidates to be the synchronous standbys.
+    def _get_replica_list(self, cluster: Cluster) -> Iterator[Tuple[int, str, str, int, bool]]:
+        """Yields candidates based on higher replay/remote_write/flush lsn.
```

Comment:
```suggestion
        """Yields candidates based on higher replay/write/flush LSN.
```

I guess you mean the fields from ``pg_stat_replication``.